MANIFEST.in

```diff
@@ -28,18 +28,20 @@ include *.sh
 include *.txt
 include .dockerignore
 include .editorconfig
-include .tx/config
-include babel.ini
-include docs/requirements.txt
 include .lgtm
+include .tx/config
```

Comment:
I think the I18N-related files should be removed from MANIFEST.in as well. Perhaps the easiest is to just run ``check-manifest -u``